content/api/ip-pools.apib

```diff
@@ -13,7 +13,8 @@ description: Manage IP pools, which allow you to group and assign dedicated IPs
     + name (string) - Name of the IP Pool
     + signing_domain (string) - DKIM-verified signing domain
     + fbl_signing_domain (string) - FBL Signing Domain for the pool. Used only for emails to domains that require DKIM for FBL.  
-    + ips (array[object]) - Sending IPs within the pool. Sending IP objects include fields `external_ip` & `hostname`
+    + ips (array[object]) - Sending IPs within the pool. Sending IP objects include fields `external_ip`,`hostname`, `auto_warmup_enabled` & `auto_warmup_stage`
+    + auto_warmup_overflow_pool (string) - IP Pool ID to be used as an overflow pool for auto IP warmup
```

Comment:
We'll want to mention that "sp_shared" can also be specified here for non-Enterprise SparkPost accounts to overflow to the SparkPost shared pool.